README.md

```diff
@@ -150,6 +150,23 @@ Other Style Guides
     console.log(a); // ReferenceError
     console.log(b); // ReferenceError
     ```
+  <a name="references--difference-let-var"></a><a name="2.4"></a>
+  - [2.4](#references--difference-let-var) Difference between `let` and `const`
+    
+    For loop with **var** keyword
+    ```javascript
+    for (var i = 1; i <= 2; i++ ) {
+      console.log(i)
+    }
+    console.log(i) // Prints 3
+    ```
+    For loop with **let** keyword
+    ```javascript
+    for (let j = 1; j <= 2; j++ ) {
+      console.log(j)
+    }
+    console.log(j) // Error
+    ```
```

Comment:
while explaining the difference is helpful, this guide forbids both `var` and loops, so i'm not sure this is the best example. Perhaps something that shows the hoisting differences - maybe just adding a `var` to the previous section?

Reply:
I will open a new PR for the same. Thank you for your assistance.

Reply:
Please do not open a PR; please repurpose this one instead.